BUMO/week-3-2/src/chat/menu/controller.js

```diff
@@ -1,11 +1,15 @@
 module.exports = class MenuController {
-    constructor(ChatService) {
+    constructor(ChatService, $scope) {
```

Comment:
we don't use $scope here. I've never shown this, so please don't use code from somewhere else.